src/components/content/monitor/Monitor.tsx

```diff
@@ -193,6 +199,7 @@ function Monitor(): JSX.Element {
                 </h3>
             </div>
             <MonitorTip type={tipType} msg={tipMessage} description={tipDescription} onRemove={onRemove} />
+
```

Comment:
in line 219, we need to show the 'loading' of the dropdown when the query is still running. Otherwise we simply show this empty dropdown. 
'Select' antd component already supports the 'loading' option. You check in credentials component for a working example. 

![image](https://github.com/eclipse-xpanse/xpanse-ui/assets/54129659/12349e97-b849-4539-8155-5d0dfd0a1f1a)